packages/dds/tree/src/test/feature-libraries/chunked-forest/uniformChunk.spec.ts

```diff
@@ -114,7 +114,7 @@ describe("uniformChunk", () => {
 	];
 
 	for (const { name: cursorName, factory } of cursorSources) {
-		describe(`${cursorName} bench`, () => {
+		describe.skip(`${cursorName} bench`, () => {
```

Comment:
why this? i don't see this failing?

Reply:
Because it's using polygonTree below -- but you're probably right, that we don't need to disable this test, but just remove it from testData which is what the basicChunk file uses. I'll remove this skip.